--- blocks.def.h.orig	2026-06-20 19:44:51.485367665 +0100
+++ blocks.def.h	2026-06-20 19:44:12.141626507 +0100
@@ -1,11 +1,12 @@
 //Modify this file to change what commands output to your statusbar, and recompile using the make command.
 static const Block blocks[] = {
 	/*Icon*/	/*Command*/		/*Update Interval*/	/*Update Signal*/
-	{"Mem:", "free -h | awk '/^Mem/ { print $3\"/\"$2 }' | sed s/i//g",	30,		0},
-
-	{"", "date '+%b %d (%a) %I:%M%p'",					5,		0},
+	{"", 		"sb-dwm",				0,		3},
+	{"", 		"sb-net",				5,		4},
+	{"", 		"sb-volume",				0,		10},
+	{"", 		"sb-clock",				60,		1},
 };
 
 //sets delimiter between status commands. NULL character ('\0') means no delimiter.
-static char delim[] = " | ";
-static unsigned int delimLen = 5;
+static char delim[] = "  ";
+static unsigned int delimLen = 2;
